docs/src/algebraic.md

```diff
@@ -38,8 +28,7 @@ For fast calculation in $\overline{\mathbb{Q}}$,
 `CalciumField` should typically be used instead (see the section
 on *Exact real and complex numbers*).
 Alternatively, to compute in a fixed subfield of $\overline{\mathbb{Q}}$,
-you may fix a generator $a$ and construct an
-Antic number field to represent $\mathbb{Q}(a)$.
+you may fix a generator $a$ and construct a number field to represent $\mathbb{Q}(a)$.
```

Comment:
I realize you are just editing existing text, so this is not a change request for this PR, but I genuinely wonder how to read this. Is `a` supposed to be an element of `CalciumField` ?! But in general were would `a` come from? Shouldn't we just say something like 

> Alternatively, one can often work in a suitable number field created via [`number_field`](@ref).